rclcpp/src/rclcpp/node_interfaces/node_graph.cpp

```diff
@@ -18,6 +18,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <algorithm>
```

Comment:
nitpick: alphabetically order the includes

Reply:
Your nitpick is my command.